mono/utils/mono-hwcap-arm.c

```diff
@@ -94,6 +94,31 @@ mono_hwcap_arch_init (void)
 	}
 
 	/* TODO: Find a way to detect features like Thumb and VFP. */
+#elif defined (_WIN32)
+    /* From MSDN:	
+     * Windows on ARM presumes that it is running on an ARMv7 architecture at all times. 
```

Comment:
Thank goodness for all this btw. Never softfloat and always simd, and not a fragmenteded instruction set.